Server/mods/deathmatch/logic/lua/CLuaModule.h

```diff
@@ -47,32 +47,36 @@ struct FunctionInfo
     RegisterModuleFunc ResourceStopped;
 };
 
-class CLuaModule : public ILuaModuleManager10
+class CLuaModule : public ILuaModuleManager
 {
 public:
     CLuaModule(CLuaModuleManager* pLuaModuleManager, CScriptDebugging* pScriptDebugging, const char* szFileName, const char* szShortFileName);
     virtual ~CLuaModule();
 
     // functions for external modules until DP2.3
-    void      Printf(const char* szFormat, ...);
-    void      ErrorPrintf(const char* szFormat, ...);
-    void      DebugPrintf(lua_State* luaVM, const char* szFormat, ...);
-    bool      RegisterFunction(lua_State* luaVM, const char* szFunctionName, lua_CFunction Func);
-    bool      GetResourceName(lua_State*   luaVM,
-                              std::string& strName);            // This function might not work if module and MTA were compiled with different compiler versions
+    void Printf(const char* szFormat, ...);
+    void ErrorPrintf(const char* szFormat, ...);
+    void DebugPrintf(lua_State* luaVM, const char* szFormat, ...);
+
+    bool        RegisterFunction(lua_State* luaVM, const char* szFunctionName, lua_CFunction Func);
+    std::string GetResourceName(lua_State* luaVM);
```

Comment:
I do feel like that the old function was better though.

Reply:
The old function caused additional copies, new one should be a better approach